tests/test_parameters.py

```diff
@@ -55,11 +55,11 @@ def run_custom_parameters_test(self, agent_policies, expected_result, delete_log
 class CustomParameterTests(BaseParameterTest):
     """Tests of custom parameters"""
 
-    def test_successfully_pass_custom_parameters(self):
-        """Test successfully setting to passsed in values"""
-        agent_policies = ["single_lp:amount_to_lp=200", "single_short:amount_to_trade=500"]
-        expected_result = [{"amount_to_lp": 200}, {"amount_to_trade": 500}]
-        self.run_custom_parameters_test(agent_policies=agent_policies, expected_result=expected_result)
+    # def test_successfully_pass_custom_parameters(self):
```

Comment:
remove dead code.  though actually why isn't this passing.  probably should ya?

Reply:
with the typo fix above, it passes :D